frontend/assets/stylesheets/components/nav/_nav.scss

```diff
@@ -9,7 +9,8 @@
  * The root component nav and >li>a selectors should not be modified directly,
  * instead use modifier classes such as "nav--columns"
  */
-.nav {
+
+@mixin navigation() {
```

Comment:
Arguably this could be called something like `unstyled-list`? Not a big deal though.